.github/actions/smoke-test/action.yml

```
@@ -25,10 +25,10 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Set up Ruby 3.2.5
+    - name: Setup Ruby
```

I think we want the verb here rather than the noun
```suggestion
    - name: Set up Ruby
```